CSS/Components/alert.css

```diff
@@ -0,0 +1,45 @@
+.alert {
+  width: 100%;
+  padding: 0.8rem 1.5rem;
+  color: #fff;
+  background-color: var(--primary-color);
+  font-size: 1rem;
+  font-weight: 500;
+  box-shadow: 0 0 3px rgba(0, 0, 0, 0.5);
+  margin: 0.7rem 0;
+  border-radius: 4px;
+}
+
+.primary {
```

Comment:
it would be better to use the class name as .alert-primary than primary. what primary does it adds primary color to everywhere you use primary as class. Though it will still work in your case, I dont feel you should keep classes that define only color inside alert.css file

Reply:
okay makes sense will work on it